lib/Db/FaceMapper.php

```diff
@@ -175,6 +198,26 @@ public function deleteUserFaces(string $userId) {
 			->execute();
 	}
 
+	/**
+	 * Unset relation beetwen faces and persons from that user in order to reset clustering
+	 *
+	 * @param string $userId User to drop fo unset relation.
+	 */
+	public function unsetPersonsRelationForUser(string $userId) {
+		$sub = $this->db->getQueryBuilder();
+		$sub->select(new Literal('1'));
+		$sub->from('facerecog_images', 'i')
+			->where($sub->expr()->eq('i.id', '*PREFIX*' . $this->getTableName() .'.image'))
```

Comment:
should you filter here by model? As a user, I would be annoyed if I do `reset` and it wipes clustering of my non-current model:)

Reply:
Ohh.. You're right!

Reply:
The same applies to delete persons, but I am seeing that we (Probably just me) ignore the consideration of the model in many PersonMapper queries. :disappointed: 

That said, there will be a future PR when implement other models. :disappointed: